### PR TITLE
add dex client troubleshooting steps

### PR DESCRIPTION
**Pull request by robinbobbitt**

### CODE REVIEW

File: troubleshooting.adoc [hunk 1]
```diff
@@ -129,3 +129,44 @@ dex-server   <none>   yourRouteSubDomain.apps.clusterurl.com    router-default.a
...
+[source,terminal]
+----
+$ oc get dexclients -n idp-mgmt-<routeSubDomain>
+NAME                                   AGE
+<managed_cluster>-<authrealm>          7m18s
+----
+You should see an entry for your managed cluster. If so, check the status to confirm the DexClient CR was successfully applied and an oauth2client was created on the Dex server.
+[source,terminal]
+----
+$ oc get dexclient  ryb-mc-nov11-authrealm-sample-github  -n idp-mgmt-testdomain -ojson | jq -r '.status.conditions'
```
Comment (chrisahl) [inline troubleshooting.adoc:148]: I think you want <managed_cluster>-<authrealm> -n idp-mgmt-<routeSubDomain>

**Review by vidyanambiar — APPROVED** [review]
/lgtm

File: troubleshooting.adoc (hunk 1 above)
Comment (robinbobbitt) [reply troubleshooting.adoc]: Thank you! Fixed.

**Pull request merged**